EIPS/eip-2124.md

```diff
@@ -0,0 +1,258 @@
+---
+eip: 2124
+title: Zero RTT netsplit on chain mismatch
+author: Péter Szilágyi (peterke@gmail.com)
+discussions-to: https://github.com/ethereum/EIPs/issues/2125
+status: Draft
+type: Standards Track
+category: Networking
+requires: 778
+created: 2019-05-03
+---
+
```

Comment:
The "Simple Summary" section is missing here.

Reply:
What's the difference between Simple Summary and the Abstract? Never understood that, imho the two things are exactly the same.

Reply:
(Will gladly add more info. just trying to understand what)

Reply:
Same here, but I think the main difference is: Simple Summary is for non-tech audience. Abstract is for a tech audience.

Reply:
Added, PTAL

Reply:
Looks good.